specification/security/resource-manager/Microsoft.Security/stable/2023-11-15/apiCollections.json

```diff
@@ -0,0 +1,473 @@
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ApiCollectionList": {
+      "type": "object",
+      "description": "Page of a list of API collections as represented by Microsoft Defender for APIs.",
+      "properties": {
+        "value": {
+          "description": "API collections in this page.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiCollection"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "ApiCollection": {
+      "type": "object",
+      "description": "An API collection as represented by Microsoft Defender for APIs.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ApiCollectionProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "ApiCollectionProperties": {
+      "type": "object",
+      "description": "Describes the properties of an API collection.",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the provisioning state of the API collection.",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "InProgress"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Succeeded"
+              },
+              {
+                "value": "Failed"
+              },
+              {
+                "value": "Canceled"
+              },
+              {
+                "value": "InProgress"
+              }
+            ]
+          }
+        },
+        "displayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the API collection."
+        },
+        "discoveredVia": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource Id of the resource from where this API collection was discovered.",
+          "format": "arm-id",
+          "x-ms-arm-id-details": {
+            "allowedResources": [
+              {
+                "type": "Microsoft.ApiManagement/service"
+              }
+            ]
+          }
+        },
+        "baseUrl": {
+          "readOnly": true,
+          "type": "string",
```

Comment:
I used uri instead of url since that's used in the same context by others and is used more frequently. Please let me know if you'd like me to stick to url

Reply:
No, uri is fine. But you should update the description to say URI so it's consistent.

Reply:
I've updated the description to say URI. Thanks!